packages/now-build-utils/src/fs/run-user-scripts.ts

```diff
@@ -39,6 +39,66 @@ export function spawnAsync(
   });
 }
 
+export function execAsync(
+  command: string,
+  args: string[],
+  opts: SpawnOptions = {}
+) {
+  return new Promise<{ stdout: string; stderr: string; code: number }>(
+    (resolve, reject) => {
+      opts.stdio = 'pipe';
+
+      let stdout: Buffer = Buffer.from('');
+      let stderr: Buffer = Buffer.from('');
+
+      const child = spawn(command, args, opts);
+
+      child.stderr!.on('data', data => {
+        stderr = Buffer.concat([stderr, data]);
+      });
+
+      child.stdout!.on('data', data => {
+        stdout = Buffer.concat([stdout, data]);
+      });
+
+      child.on('error', reject);
+      child.on('close', (code, signal) => {
+        if (code !== 0) {
+          return reject(
+            new Error(
+              `Program "${command}" exited with non-zero exit code ${code} ${signal}.`
+            )
+          );
+        }
+
+        return resolve({
+          code,
+          stdout: stdout.toString(),
+          stderr: stderr.toString(),
+        });
+      });
+    }
+  );
+}
+
+export function spawnCommand(command: string, options: SpawnOptions = {}) {
+  if (process.platform === 'win32') {
+    return spawn('cmd.exe', ['/C', command], options);
+  }
+
+  return spawn('sh', ['-c', command], options);
```

Comment:
The only thing I'm not sure about here is if we need to be more specific and use `bash` instead of `sh` since its not really known if the user's choice of shell will have a `-c` option.

cc @TooTallNate who might know

Reply:
`sh` is fine. You could argue that it's not known if `bash` is even available on the user's machine (i.e. Alpine Linux doesn't come with bash by default).